tastytrade/instruments.py

```diff
@@ -91,6 +90,26 @@ def get_cryptocurrency(cls, session: Session, symbol: str) -> 'Cryptocurrency':
         return cls.from_dict(data)
 
 
+@dataclass
+class TickSizes:
+    value: str
+    threshold: Optional[str] = None
+    symbol: Optional[str] = None
+
+
+@dataclass
+class Pagination:
```

Comment:
So this isn't being used anymore, right?

Reply:
right

Reply:
Actually I already removed it :D